compiler/optimizations/loopInvariantCodeMotion.cpp

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright 2017 Advanced Micro Devices, Inc.
```

Comment:
I'm personally rather apathetic about these types of things, but our legal team originally wanted me to basically put the same entire generic Apache license text you have with our copyright for all the files I modify. I told them that it looked pretty ridiculous to duplicate all that text, so they agreed just adding that single line on top of your text was sufficient. If you guys want something different, I can circle back with AMD legal and see what they think.